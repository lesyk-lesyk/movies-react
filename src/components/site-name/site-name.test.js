import React from 'react';
import { SiteName } from './index';
import renderer from 'react-test-renderer';

describe('SiteName', () => {
  it('renders', () => {
    const tree = renderer
      .create(<SiteName />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
