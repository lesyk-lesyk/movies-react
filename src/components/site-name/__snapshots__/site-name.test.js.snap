// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SiteName renders 1`] = `
<span
  className="site-name"
>
  netflixroulette
</span>
`;
